samples/snippets/create_custom_entry.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_custom_entry(project_id, entry_group_id, entry_id, tag_template_id):
+    """Creates a custom entry within an entry group."""
+    # [START data_catalog_create_custom_entry]
+    # -------------------------------
+    # Import required modules.
+    # -------------------------------
+    from google.api_core.exceptions import NotFound, PermissionDenied
+    from google.cloud import datacatalog_v1
+
+    # -------------------------------
+    # TODO: Set these values before running the sample.
+    # -------------------------------
+    # project_id = 'my-project'
+    # entry_group_id = 'onprem_entry_group'
+    # entry_id = 'onprem_entry_id'
+    # tag_template_id = 'onprem_tag_template'
+
+    # -------------------------------
+    # Currently, Data Catalog stores metadata in the
+    # us-central1 region.
```

Comment:
I think this comment is out-of-date. We should link to the supported regions. https://cloud.google.com/data-catalog/docs/concepts/regions